### ISSUE TITLE
Accessibility: YouTube embeds lack title

### ISSUE THREAD

**Issue by aduth** [GSA/centers-of-excellence#372]
YouTube embed `<iframe>` elements lack a `title`, which is used by users to determine which frame to enter and explore in detail.

_Note:_ This error was captured by automated testing, using the [aXe browser extension](https://www.deque.com/axe/browser-extensions/). See also ["Frames must have title attribute"](https://dequeuniversity.com/rules/axe/3.3/frame-title).

**Affected pages:**

- [x] https://coe.gsa.gov/2018/05/18/cc-update-1.html
- [x] https://coe.gsa.gov/connect/contact-us.html
- [x] https://coe.gsa.gov/2018/06/09/coe-update-1.html

**WCAG Success Criteria (AA):** [2.4.1: Bypass Blocks](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html)

**Relevant resources:**

https://dequeuniversity.com/rules/axe/3.3/frame-title

>Screen reader users rely on a frame title to describe the contents of the frame. Navigating through frame and iframe elements quickly becomes difficult and confusing for users of this technology if the markup does not contain a title attribute.

https://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140408/H64

>The objective of this technique is to demonstrate the use of the title attribute of the frame or iframe element to describe the contents of each frame. This provides a label for the frame so users can determine which frame to enter and explore in detail.

https://developer.paciellogroup.com/blog/2010/11/using-the-html-title-attribute/

>Examples of title attribute use that are USEFUL: [...] Labeling frame or iframe elements:

https://adrianroselli.com/2017/08/inaccessible-youtube-embeds.html

>Most of us in the accessibility biz (do we call it that? I think we call it that…) already know that YouTube’s default embed code is problematic. Specifically, the fact that the `<iframe>` does not have a `title` attribute is an automatic WCAG 2.0 AA failure.

https://www.davidmacd.com/blog/is-title-attribute-on-iframe-required-by-wcag.html

**Remediation Guidance:**

These YouTube embed codes are provided by YouTube, but are manually pasted into the contents of the pages. It could be possible to manually add the `title` attribute.
